docs/build/isc/v1.0.0-rc.6/docs/how-tos/core-contracts/nft/get-nft-metadata.md

```diff
@@ -0,0 +1,153 @@
+---
+description: How to get NFT metadata on L1
+image: /img/logo/WASP_logo_dark.png
+tags:
+  - NFT
+  - EVM
+  - how-to
+---
+import GetNftMetadata from '../../../_partials/how-tos/token/_get-nft-metadata.md';
+
+# Get NFT Metadata
+
+This guide explains how to utilize the `getIRC27NFTData` function within a smart contract to fetch information about a specific IRC27 NFT on the IOTA Network. The function resides in the ISCSandbox contract and returns details pertaining to the requested NFT.
+
+## Understanding the `getIRC27NFTData` Function
+
+The `getIRC27NFTData` function retrieves metadata for an IRC27 NFT based on its identifier. IRC27 is a series of standards to support interoperable and universal NFT systems throughout the IOTA ecosystem.
+
+## Function Signature:
+```solidity
+function getIRC27NFTData(bytes memory nftID) public view returns (IRC27NFT memory irc27NftData);
+```
+## Parameters:
+`NFTID id`: This parameter represents the unique identifier of the IRC27 NFT you intend to retrieve information about.
+## Return Value:
+The function returns a data structure of type IRC27NFT. This structure incorporates two elements:
+*  `nft`: This element provides details regarding the underlying on-chain NFT. It's of type `ISCNFT`.
+*  `metadata`: This element offers information specific to the IRC27 standard. It's of type `IRC27NFTMetadata`.
```

Comment:
````suggestion
### Function Signature:

```solidity
function getIRC27NFTData(bytes memory nftID) public view returns (IRC27NFT memory irc27NftData);
```

### Parameters:

`NFTID id`: This parameter represents the unique identifier of the IRC27 NFT you intend to retrieve information about.

### Return Value:

The function returns a data structure of type IRC27NFT. This structure incorporates two elements:
*  `nft`: This element provides details regarding the underlying on-chain NFT. It's of type `ISCNFT`.
*  `metadata`: This element offers information specific to the IRC27 standard. It's of type `IRC27NFTMetadata`.
````

This probably belongs in the refs, though I think they are autogenerated.

Reply:
I made the changes, but like you said it belongs in the refs, let me know if I should take this section out